src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/[[...slug]]/page.tsx

```diff
@@ -51,6 +51,7 @@ type Props = {
   };
   searchParams: {
     nimbus_web_preview?: string;
+    dialog?: "subscriptions";
```

Comment:
The query param name and value are TBD. I’ve asked about the final convention [here](https://mozilla-hub.atlassian.net/browse/MNTOR-3619?focusedCommentId=938349).